e2e/libs/replica/crd.py

```diff
@@ -33,10 +33,7 @@ def delete(self, volume_name, node_name):
             logging(
                 f"Deleting volume {volume_name} on node {node_name} replicas")
 
-        resp = self.get(volume_name, node_name)
-        assert resp != "", f"failed to get replicas"
```

Comment:
Regarding the removal of `assert resp != "", f"failed to get replicas"`, could you explain the main reasoning behind removing this step? I don't quite understand it.